app/forms/waste_carriers_engine/location_form.rb

```diff
@@ -4,9 +4,11 @@ module WasteCarriersEngine
   class LocationForm < BaseForm
     attr_accessor :location
 
+    validates :location, "defra_ruby/validators/location": { allow_overseas: true }
+
     def initialize(transient_registration)
       super
-      self.location = @transient_registration.location
+      self.location = transient_registration.location
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.location = transient_registration.location
```